eng/testing/WasmRunnerAOTTemplate.sh

```diff
@@ -35,8 +35,10 @@ if [[ "$XHARNESS_COMMAND" == "test" ]]; then
 	if [[ -z "$JS_ENGINE" ]]; then
 		if [[ "$SCENARIO" == "WasmTestOnNodeJS" || "$SCENARIO" == "wasmtestonnodejs" ]]; then
 			JS_ENGINE="--engine=NodeJS"
+			JS_ENGINE_ARGS="$JS_ENGINE_ARGS --engine-arg=--experimental-wasm-simd"
 		else
 			JS_ENGINE="--engine=V8"
+			JS_ENGINE_ARGS="$JS_ENGINE_ARGS --engine-arg=--experimental-wasm-simd"
```

Comment:
are we going to keep any lane that tests what happens if simd is disabled? or are we just going to require wasm simd for P3 and later, period. i don't have a good sense of how widespread browsers without wasm simd support are and whether we need to have automated test coverage for what happens

Reply:
I will add this test in folloup PR.

Reply:
https://github.com/dotnet/runtime/pull/82845